Code/Editor/EditorViewportWidget.cpp

```diff
@@ -3134,25 +2898,16 @@ bool EditorViewportWidget::ShouldPreviewFullscreen() const
         return false;
     }
 
-    // Doesn't work with split layout
-    if (layout->GetLayout() != EViewLayout::ET_Layout0)
-    {
-        return false;
-    }
+    // Doesn't work with split layout (TODO: figure out why and make it work)
+    if (layout->GetLayout() != EViewLayout::ET_Layout0) { return false; }
```

Comment:
Happy to take the TODO but our coding style doesn't allow for one-liner ifs, please restore the previous formatting.

Reply:
Resolved in 8baaba0e85